app/components/TaskList.tsx

```diff
@@ -145,9 +271,19 @@ const TaskList: React.FC<Props> = ({ projectRevision }) => {
         }
 
         div {
-          width: 400px;
           background-color: #e5e5e5;
         }
+        .row {
+          display: flex;
+        }
+        .link {
+          flex: 50%;
+        }
+        .status {
+          padding-right: 10px;
+          text-align: right;
+          flex: 40%;
+        }
```

Comment:
```suggestion
        ul > li {
          display: flex;
          justify-content: space-between;
        }
```

would this work?

Reply:
This causes padding/alignment issues.  I tried a few approaches, flex with row/col is the only way I got it to work
<img width="367" alt="Screen Shot 2022-04-26 at 12 41 39 PM" src="https://user-images.githubusercontent.com/9407864/165379423-42997ea4-f737-4879-8230-8fd678e65c43.png">
.